### ISSUE TITLE
Crash on OSX - TypeError: Cannot read property 'debug' of undefined

### ISSUE THREAD

**Issue by elribonazo** [dubocr/homebridge-tahoma#117]
Hi,

Thanks for taking time to read the issue.
Well basically I've got it configured and every X, I get 

```
TypeError: Cannot read property 'debug' of undefined
    at Timeout.requestWithLogin [as _onTimeout] (/usr/local/lib/node_modules/homebridge-tahoma/overkiz-api.js:228:22)
    at ontimeout (timers.js:438:13)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
```

Maybe there's a package dependency with homebridge version?

**Comment by reg10** [comment]
Same issue here, after a upgrade noting works any more and getting the same response.
spanning the refreshperiode didn't change a bit

/root/.homebridge/.env not found.                                                                                                                                                                                            
Default env variables will be used.                                                                                                                                                                                          
/root/.homebridge/package.json not found.                                                                                                                                                                                    
/root/.homebridge/install.sh not found.                                                                                                                                                                                      
Failed to start message bus: Failed to bind socket "/var/run/dbus/system_bus_socket": Address already in use                                                                                                                 
[8/7/2019, 7:56:45 PM] Loaded config.json with 0 accessories and 1 platforms.                                                                                                                                                
[8/7/2019, 7:56:45 PM] ---                                                                                                                                                                                                   
[8/7/2019, 7:56:45 PM] Loaded plugin: homebridge-tahoma                                                                                                                                                                      
homebridge-tahoma API version: 2.4                                                                                                                                                                                           
[8/7/2019, 7:56:45 PM] Registering platform 'homebridge-tahoma.Tahoma'                                                                                                                                                       
[8/7/2019, 7:56:45 PM] ---                                                                                                                                                                                                   
[8/7/2019, 7:56:45 PM] Loading 1 platforms...                                                                                                                                                                                
[8/7/2019, 7:56:45 PM] [Connexoon RTS] Initializing Tahoma platform...                                                                                                                                                       
[8/7/2019, 7:56:45 PM] [Connexoon RTS] Fetch accessories...                                                                                                                                                                  
[8/7/2019, 7:56:45 PM] Loading 0 accessories...                                                                                                                                                                              
[8/7/2019, 7:57:05 PM] [Connexoon RTS] RESP : undefined                                                                                                                                                                      
[8/7/2019, 7:57:05 PM] [Connexoon RTS] Unable to login: Error: getaddrinfo EAI_AGAIN tahomalink.com tahomalink.com:443                                                                                                       
[8/7/2019, 7:57:05 PM] [Connexoon RTS] Retry 1/3                                                                                                                                                                             
[8/7/2019, 7:57:05 PM] [Connexoon RTS] RESP : undefined                                                                                                                                                                      
[8/7/2019, 7:57:05 PM] [Connexoon RTS] Unable to login: Error: getaddrinfo EAI_AGAIN tahomalink.com tahomalink.com:443                                                                                                       
[8/7/2019, 7:57:05 PM] [Connexoon RTS] Retry 2/3                                                                                                                                                                             
[8/7/2019, 7:57:06 PM] TypeError: Cannot read property 'debug' of undefined                                                                                                                                                  
    at Timeout.requestWithLogin [as _onTimeout] (/usr/local/lib/node_modules/homebridge-tahoma/overkiz-api.js:228:22)                                                                                                        
    at ontimeout (timers.js:427:13)                                                                                                                                                                                          
    at tryOnTimeout (timers.js:289:5)                                                                                                                                                                                        
    at listOnTimeout (timers.js:252:5)                                                                                                                                                                                       
    at Timer.processTimers (timers.js:212:10)                                                                                                                                                                                
[8/7/2019, 7:57:06 PM] Got SIGTERM, shutting down Homebridge...                                                                                                                                                              
[8/7/2019, 7:57:06 PM] TypeError: Cannot read property 'debug' of undefined                                                                                                                                                  
    at Timeout.requestWithLogin [as _onTimeout] (/usr/local/lib/node_modules/homebridge-tahoma/overkiz-api.js:228:22)                                                                                                        
    at ontimeout (timers.js:427:13)                                                                                                                                                                                          
    at tryOnTimeout (timers.js:289:5)                                                                                                                                                                                        
    at listOnTimeout (timers.js:252:5)                                                                                                                                                                                       
    at Timer.processTimers (timers.js:212:10)

**Comment by dubocr** [comment]
Hi,

This is fully fixed in 0.3.11

Thanks

**Comment by elribonazo** [comment]
@dubocr lots of thanks!
